modules/setting/setting.go

```diff
@@ -222,12 +222,16 @@ func loadCommonSettingsFrom(cfg ConfigProvider) {
 
 	loadOAuth2From(cfg)
 	loadSecurityFrom(cfg)
-	loadAttachmentFrom(cfg)
+	if err := loadAttachmentFrom(cfg); err != nil {
+		log.Fatal("loadAttachmentFrom: %v", err)
+	}
 	loadLFSFrom(cfg)
 	loadTimeFrom(cfg)
 	loadRepositoryFrom(cfg)
 	loadPictureFrom(cfg)
-	loadPackagesFrom(cfg)
+	if err := loadPackagesFrom(cfg); err != nil {
+		log.Fatal("loadPackagesFrom: %v", err)
+	}
```

Comment:
Will this `loadCommonSettingsFrom` be filled with a lot of `if { ... }`? It seems less clear than before.

Reply:
how about using a loop like this?
```GO
	type initItem struct {
		Name string
		Func func(cfg ConfigProvider) error
	}

	// ....

	var initList []initItem

	for _, init := range initList {
		if err := init.Func(cfg); err != nil {
			log.Fatal("%s: %v", init.Name, err)
		}
	}
```

Reply:
Done.